egs/cifar/v1/image/ocr/make_features.py

```diff
@@ -144,21 +179,25 @@ def get_scaled_image_aug(im, mode='normal'):
         im = misc.imread(image_path)
         if args.fliplr:
             im = np.fliplr(im)
-        if args.augment:
-            im_aug = get_scaled_image_aug(im, aug_setting[1])
-        else:
-            im_aug = get_scaled_image_aug(im, aug_setting[0])
-        im_horizontal_padded = horizontal_pad(im_aug, allowed_lengths)
-        if im_horizontal_padded is None:
+        if args.augment_type == 'no_aug' or 'random_shift':
+            im = get_scaled_image_aug(im, aug_setting[0])
```

Comment:
It's not clear to me why the `aug_setting` variable is used here, I think it would be more clear if you directly used 'normal', etc.